core/dbt/parser/common.py

```diff
@@ -0,0 +1,222 @@
+from dbt.contracts.graph.unparsed import (
+    HasColumnProps,
+    UnparsedColumn,
+    UnparsedNodeUpdate,
+    UnparsedMacroUpdate,
+    UnparsedAnalysisUpdate,
+    UnparsedExposure,
+    UnparsedModelUpdate,
+)
+from dbt.contracts.graph.unparsed import NodeVersion, HasColumnTests, HasColumnDocs
+from dbt.contracts.graph.nodes import (
+    UnpatchedSourceDefinition,
+    ColumnInfo,
+    ColumnLevelConstraint,
+    ConstraintType,
+)
+from dbt.parser.search import FileBlock
+from typing import List, Dict, Any, TypeVar, Generic, Union, Optional
+from dataclasses import dataclass
+from dbt.exceptions import DbtInternalError, ParsingError
+
+
+def _trimmed(inp: str) -> str:
```

Comment:
nit: maybe we drop the `_` prefix given this method is imported outside the file